aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -75,7 +75,7 @@ class DeleteModal extends Component {
               </span>
               <FormattedMessage
                 id='delete.modal.all.submissions-2'
-                defaultMessage={`this ${this.props.objectType} (Submissions)`}
+                defaultMessage='this pipeline (Submissions)'
```

Comment:
Could it be "contract"?

Reply:
No, only pipelines/mappingsets have submissions

Reply:
Right!